tour/basic-read-write.md

```diff
@@ -1,5 +1,67 @@
 ---
-title: Basic Reading & Writing
+title: Writing and Reading
 ---
+Accumulo is a big data key/value store.  Writing data to Accumulo is flexible and fast.  Like any database, Accumulo stores
+data in tables and rows.  Each row in an Accumulo table can hold many key/value pairs.  
 
-Talk about reading and writing.
+1. Start by connecting to Mini Accumulo and create a table called "superheroes".  For now, connect as the root user.
+```java
+Connector conn = mac.getConnector("root", "tourguide");
+conn.tableOperations().create("superheroes");
+```
+
+2. Create a Mutation object for row1
+```java
+Mutation mutation = new Mutation("row1");
+```
+A Mutation is an object that holds all changes to a row in a table.  Each row has a unique row ID.
+ 
+3. Create key/value pairs for Batman.
+```java
+mutation.put("name", "", "Batman");
+mutation.put("real-name", "", "Bruce Wayne");
+mutation.put("wearsCape?", "", "true");
+mutation.put("flies?", "", "false");
+```
+Every Mutation in Accumulo is atomic. This means that all the changes to a single row will happen at once. The Mutation
+object conveniently allows us to put all the changes for the row in one spot. 
+
+4. Create a BatchWriter to the superhero table and add your mutation to it.
```

Comment:
Could say " Write the mutation to the superhero table."  The fact that a batch writer is used to do is self evident from the code.